### PR TITLE
Update customer portal links to account for Chargebee companies

### PR DESCRIPTION
**Pull request by fjvallarino**
Changes to stage.js will be reverted before merging

@Rise-Vision/apps please review

### CODE REVIEW

File: web/partials/editor/store-products-modal.html [hunk 1]
```diff
@@ -140,7 +140,6 @@ <h4>{{'editor-app.storeProduct.templates.blank' | translate}}</h4>
                 <div class="row">
                   <div class="col-xs-8">
                     <h4 id="productName">{{product.name | characters: 25}}</h4>
-                    <a ng-hide="factory.search.category === 'Templates'" ng-href="{{storeUrl}}/product/{{product.productId}}/" target="_blank" ng-click="$event.stopPropagation()" translate="editor-app.storeProduct.viewInStore"></a>
```
Comment (alex-deaconu) [inline web/partials/editor/store-products-modal.html]: @fjvallarino we should clean out this i18n string. Thanks.

Comment (fjvallarino) [reply web/partials/editor/store-products-modal.html]: @alex-deaconu removed from Common Header

**Comment by fjvallarino** [comment]
After merging Store and Common Header, config files and dependencies will be updated/reverted

@Rise-Vision/apps please review

File: web/index.html [hunk 2]
```diff
@@ -59,6 +59,8 @@
   <div ui-view></div>
 
   <!-- dependencies -->
+  <script src="https://js.chargebee.com/v2/chargebee.js"></script>
```
Comment (alex-deaconu) [inline web/index.html]: @fjvallarino any way to make this an async script load? ie. make this load at runtime? Thanks.

Comment (fjvallarino) [reply web/index.html]: @alex-deaconu what about using the ```async``` tag? https://www.w3schools.com/Tags/att_script_async.asp

Comment (alex-deaconu) [reply web/index.html]: @fjvallarino that works for me. Do you think it would help? Thanks.

Comment (olegrise) [reply web/index.html]: @alex-deaconu @fjvallarino interesting idea. I never used async before, but it seems that in this situation it's a good fit.

File: web/scripts/config/test.js [hunk 3]
```diff
@@ -43,6 +43,7 @@
     .value('APPS_URL', '')
     .value('ENV_NAME', 'TEST')
     .value('OAUTH_TOKEN_PROVIDER_URL', 'https://services-stage.risevision.com/oauthtokenprovider/')
-    .value('OAUTH_PUBLIC_KEY', 'EJMI-lB9hB55OYEsYmjXDNfRGoY');
+    .value('OAUTH_PUBLIC_KEY', 'EJMI-lB9hB55OYEsYmjXDNfRGoY')
+    .value('CHARGEBEE_INSTANCE', 'risevision-test');
```
Comment (olegrise) [inline web/scripts/config/test.js]: @fjvallarino is this used anywhere?

Comment (fjvallarino) [reply web/scripts/config/test.js]: @olegrise no, I missed that file when renaming/adding the new constants. I'll update it when I'm back at home

**Comment by fjvallarino** [comment]
@olegrise sorry to bug you again. I need review/approval on this one. Thanks!

**Pull request merged**